netket/models/gaussian.py

```diff
@@ -0,0 +1,35 @@
+import flax.linen as nn
+import jax.numpy as jnp
+
+from netket.nn.initializers import normal
+from netket.utils.types import DType, Array, NNInitFunc
+
+
+class Gaussian(nn.Module):
+    r"""Multivariate Gaussain function with mean 0
+    and covariance matrix :math:`\Sigma_{ij}`:
+    :math:`\Psi(x) = \exp(\sum_{ij} x_i \Sigma_{ij} x_j)`.
+    The :math:`\Sigma_{ij} = AA^T` matrix is stored as
+    non-positive definite matrix A.
+    """
+
```

Comment:
Its good customs to have docstring have a first short line or two and then the rest.
This is used as a preview in the doc system,

```suggestion
    r"""
    Multivariate Gaussain function with mean 0 and parametrised covariance matrix
    :math:`\Sigma_{ij}`.
    
    The wavefunction is given by the formula: :math:`\Psi(x) = \exp(\sum_{ij} x_i \Sigma_{ij} x_j)`.
    The (positive definite) :math:`\Sigma_{ij} = AA^T` matrix is stored as
    non-positive definite matrix A.
    """

```